src/main/java/com/api/service/NetworkingService.java

```diff
@@ -0,0 +1,50 @@
+package com.api.service;
+
+import com.api.entity.SX_Rocket;
+import com.api.utility.UriResolver;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.net.URI;
+
+@Service
+@PropertySource("classpath:application.properties")
+public class NetworkingService {
+
+    @Value("${rocket.api.base-url}")
+    private String getBaseUrl;
```

Comment:
the base should be the www.google.com and you need to separate the paths in roder to build the easily, in case you will have same base with multiple paths ({base_url} /rocket or /launches or /starts) you will need to resolve them dinamically